v4/source/users.yaml

```diff
@@ -1656,6 +1656,12 @@
           required: true
           schema:
             type: string
+        - name: _
+          in: query
+          description: unknown
+          required: false
+          schema:
+            type: number
```

Comment:
What's this section for?

Reply:
![image](https://user-images.githubusercontent.com/99795090/154674751-50f6f497-1213-4b6d-9b1b-8000ea1f1d00.png)
Goal is to describe in the Swagger this type of call that can actually be sent by MM client

Reply:
>Goal is to describe in the Swagger this type of call that can actually be sent by MM client 

Let's try to document the actual meaning of the parameter instead of just marking it as "unknown". 

Looking at the webapp code, https://github.com/mattermost/mattermost-webapp/blob/master/packages/mattermost-redux/src/client/client4.ts#L890

It seems the value of `_` is the `lastPictureUpdate`, but it isn't used on the server side. It's just a dummy parameter to get around any caching. 

I'm not sure if this needs to be documented in the API. 

@koox00 @Willyfrog what do you guys think?

Reply:
if it is a valid parameter, I'd document it, if only to indicate how to use it.

Reply:
I would say that we probably shouldn't document it, since it has nothing to do with the API,
i.e it's not an accepted parameter, and might give the impression that the server actually consumes it and does something when it's passed, I'm 3/5.

@fromhro is that a blocker for your use case?

Reply:
@koox00 As there are some requests using this parameter on our end we would need to have it documented in the Swagger file. Otherwise our WAF policies won't allow the request using this parameter.

Reply:
Lets change:
```
description: unknown
```
to
```
description: Not used by the server. Clients can pass in the last picture update time of the user to potentially take advantage of caching,
```

Reply:
I did the modification